code/modules/mob/living/carbon/human/human.dm

```diff
@@ -41,6 +40,15 @@
 		return
 	mob_mood = new /datum/mood(src)
 
+/// This proc is for holding effects applied when a mob is missing certain organs
+/// It is called very, very early in human init because all humans innately spawn with no organs and gain them during init
+/// Gaining said organs removes these effects
+/mob/living/carbon/human/proc/setup_no_organ_effects()
```

Comment:
could be something like `setup_organless_effects` as that seems clearer to me but whatever